metaci/release/tests/test_utils.py

```diff
@@ -18,15 +20,36 @@ def test_send_release_webhook(mocked_responses, mocker, transactional_db):
     mocked_responses.add(
         "POST",
         "https://webhook/release/",
-        json={"success": True, "implementationSteps": ["a2dAG0000000"], "id": "2"},
+        json={"success": True, "implementationSteps": ["2", "1", "3"], "id": "2"},
```

Comment:
This test is for a scenario with 2 implementation steps so we should only be expecting 2 ids to be returned, right?

Reply:
I wanted to check that the proper values were assigned to the implementation steps with the roles given.

Reply:
@jofsky I see 2 implementation steps set up on the release and 3 ids in the response. Why are they different?